python/activity_stream/activity_stream.py

```diff
@@ -372,6 +374,18 @@ def _set_notes_are_selectable(self, state):
         _get_notes_are_selectable,
         _set_notes_are_selectable,
     )
+
+    def _get_attachments_filter(self):
+        """
+        If set to a compiled regular expression, attachment file names that match
+        will be filtered OUT and NOT shown.
+        """
+        return self._attachments_filter
+
+    def _set_attachments_filter(self, regex):
```

Comment:
For properties, sphinx pulls docstrings from the getter and ignore the setter. I'll add a comment to the setter for people reading the code, and will expand the getter docstring a bit.